### DISCUSSION TITLE
Sumatra 3.3 fails

### DISCUSSION THREAD

**Discussion by seikor** [sumatrapdfreader/sumatrapdf#2374] [General]
Hello!

the actual sumatraPDF version 3.3 fails with view of some PDF-documents which will be show in 3.2.
There are boxes instead of letters so I think there are no fonts inside. 
All other viewers can view this data.

Who can help?
![signal-2022-02-24-143431_001](https://user-images.githubusercontent.com/100368070/155558766-48fa91b8-760c-49c0-8c1c-ca3ca96b7e5b.jpeg)

**Comment by GitHubRulesOK** [comment]
Yes its potentially a poor font definition problem so you need a different version

3.1.2 was the most stable fontwise and [3.4 pre-release](https://www.sumatrapdfreader.org/prerelease) has many font fixes, so suggest you try current portable pre-release first (most likely it is a fixed problem), but if 3.4PR still cant handle the font definitions a sample is needed for analysis and fix.

**Comment by seikor** [comment]
I took 3.2 its functionate. 
3.4 pre-relase do it also correctly, so I wait for fixed 3.4
The bug ist obviously only in 3.3.
Thanks!

**Comment by michael9dk** [comment]
Bug verified in v3.3.3 64bit.

I reverted back to v3.2

**Comment by hueldoeu** [comment]
i just wanted to say that somebody made an app in microsoft store for sumatraPDF https://www.microsoft.com/de-de/p/all-ebooks-reader-for-kindle-book-formats/9p7jt9p0x2dj?ranMID=46131&ranEAID=KQ6bJ8shQ6c&ranSiteID=KQ6bJ8shQ6c-YCIFhGx9f1B9sEJejvzBDg&epi=KQ6bJ8shQ6c-YCIFhGx9f1B9sEJejvzBDg&irgwc=1&OCID=AID2200057_aff_7806_1243925&tduid=%28ir__s9fri6gjg0kf6xfqm99hkpgkpf2xtyvvet3pk6nw00%29%287806%29%281243925%29%28KQ6bJ8shQ6c-YCIFhGx9f1B9sEJejvzBDg%29%28%29&irclickid=_s9fri6gjg0kf6xfqm99hkpgkpf2xtyvvet3pk6nw00&activetab=pivot:overviewtab
